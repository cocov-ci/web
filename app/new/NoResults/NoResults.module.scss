.noResults {
  margin-top: 30px;
}
