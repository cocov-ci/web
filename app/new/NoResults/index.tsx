import { Ghost } from 'lucide-react'

import Alert from 'app/common/Alert'

import styles from './NoResults.module.scss'

const Empty = () => {
  return (
    <Alert
      className={styles.noResults}
      description="Your search did not yield results. Check if your spelling is correct, or try using other terms."
      icon={Ghost}
      title="No results"
    />
  )
}

export default Empty
