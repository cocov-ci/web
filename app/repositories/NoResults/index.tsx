import { Ghost } from 'lucide-react'

import Alert from 'app/common/Alert'

const Empty = () => {
  return (
    <Alert
      description="Your search did not yield results. Check if your spelling is correct, or try using other terms."
      icon={Ghost}
      title="No results"
    />
  )
}

export default Empty
