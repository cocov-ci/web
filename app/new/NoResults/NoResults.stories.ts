import type { Meta, StoryObj } from '@storybook/react'

import NoResults from './index'

const meta: Meta<typeof NoResults> = {
  title: 'Repositories/New/NoResults',
  component: NoResults,
  tags: ['autodocs'],
}

export default meta

type Story = StoryObj<typeof NoResults>

export const Default: Story = {
  args: {},
}
