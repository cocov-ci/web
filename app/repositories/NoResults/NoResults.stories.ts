import type { Meta, StoryObj } from '@storybook/react'

import NoResults from './index'

const meta: Meta<typeof NoResults> = {
  title: 'Repositories/NoResults',
  component: NoResults,
  tags: ['autodocs'],
}

export default meta

type Story = StoryObj<typeof NoResults>

export const Default: Story = {
  args: {},
}
