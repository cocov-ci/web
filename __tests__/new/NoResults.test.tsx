import { render, screen } from '@testing-library/react'

import NoResults from 'app/new/NoResults'

import '@testing-library/jest-dom'

describe('new/NoResults', () => {
  it('renders NoResults', () => {
    const { container } = render(<NoResults />)

    expect(screen.getByText('No results')).toBeVisible()
    expect(container.querySelector('svg')).toBeVisible()
  })

  it('renders NoResults snapshots', () => {
    const { container } = render(<NoResults />)

    expect(container).toMatchSnapshot()
  })
})
