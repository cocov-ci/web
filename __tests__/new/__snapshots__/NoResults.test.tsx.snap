// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`new/NoResults renders NoResults snapshots 1`] = `
<div>
  <div
    class="alert noResults"
  >
    <div
      class="box"
    >
      <svg
        class="icon"
        fill="none"
        height="40"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="2"
        viewBox="0 0 24 24"
        width="40"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M9 10h.01"
        />
        <path
          d="M15 10h.01"
        />
        <path
          d="M12 2a8 8 0 0 0-8 8v12l3-3 2.5 2.5L12 19l2.5 2.5L17 19l3 3V10a8 8 0 0 0-8-8z"
        />
      </svg>
      <div
        class="content"
      >
        <h2>
          No results
        </h2>
        <h3>
          Your search did not yield results. Check if your spelling is correct, or try using other terms.
        </h3>
      </div>
    </div>
  </div>
</div>
`;
